From: olh@suse.de
Subject: [PATCH] ftape syntax error

2.6.11-rc2-bk5 introduces two syntax errors under drivers/char/ftape.  The following
patch replaces ");" at the end of two printks which were accidentally removed.

thanks,
-serge

Signed-off-by: Serge Hallyn <serue@us.ibm.com>

Index: linux-2.6.10/drivers/char/ftape/compressor/zftape-compress.c
===================================================================
--- linux-2.6.10.orig/drivers/char/ftape/compressor/zftape-compress.c	2005-01-27 10:25:52.000000000 -0600
+++ linux-2.6.10/drivers/char/ftape/compressor/zftape-compress.c	2005-01-27 11:10:42.750424360 -0600
@@ -1172,7 +1172,7 @@ int zft_compressor_init(void)
         if (TRACE_LEVEL >= ft_t_info) {
 		printk(
 KERN_INFO "(c) 1997 Claus-Justus Heine (claus@momo.math.rwth-aachen.de)\n"
-KERN_INFO "Compressor for zftape (lzrw3 algorithm)\n"
+KERN_INFO "Compressor for zftape (lzrw3 algorithm)\n");
         }
 #else /* !MODULE */
 	/* print a short no-nonsense boot message */
Index: linux-2.6.10/drivers/char/ftape/lowlevel/ftape-init.c
===================================================================
--- linux-2.6.10.orig/drivers/char/ftape/lowlevel/ftape-init.c	2005-01-27 10:25:52.000000000 -0600
+++ linux-2.6.10/drivers/char/ftape/lowlevel/ftape-init.c	2005-01-27 11:11:12.683873784 -0600
@@ -72,7 +72,7 @@ static int __init ftape_init(void)
 KERN_INFO "(c) 1993-1996 Bas Laarhoven (bas@vimec.nl)\n"
 KERN_INFO "(c) 1995-1996 Kai Harrekilde-Petersen (khp@dolphinics.no)\n"
 KERN_INFO "(c) 1996-1997 Claus-Justus Heine (claus@momo.math.rwth-aachen.de)\n"
-KERN_INFO "QIC-117 driver for QIC-40/80/3010/3020 floppy tape drives\n"
+KERN_INFO "QIC-117 driver for QIC-40/80/3010/3020 floppy tape drives\n");
         }
 #else /* !MODULE */
 	/* print a short no-nonsense boot message */

-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/

